model/org.eclipse.birt.report.model/src/org/eclipse/birt/report/model/elements/rom.def

```diff
@@ -2000,7 +2000,11 @@
       </Property>
 	  <!-- docx (wpml): add empty paragraph at the end of list table cell						-->
       <Property canInherit="false" displayNameID="Element.ReportDesign.Emitter.word.addEmptyParagraphForListCell" name="wordAddEmptyParagraphForListCell" runtimeSettable="true" since="4.18" type="boolean" >
-		<Default>false</Default>
+		<Default>true</Default>
+      </Property>
+	  <!-- docx (wpml): add empty paragraph at the end of table cell						-->
+      <Property canInherit="false" displayNameID="Element.ReportDesign.Emitter.word.addEmptyParagraphForTableCell" name="wordAddEmptyParagraphForTableCell" runtimeSettable="true" since="4.18" type="boolean" >
```

Comment:
since 4.19? Do we need to increase the rom version?

Reply:
Version number changed at rom and increased the report version number.